lib/gax.js

```diff
@@ -58,6 +58,9 @@
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @property {number} maxRetries - the maximum number of retrying attempts that
```

Comment:
I want to keep this option private, so that users aren't accessing it at the GAPIC layer. That way we contain the disruption to just the APIs with hand-written layers in case we decide to remove this in v2.